report/src/main/webapp/app/components/keyboardShortcuts.directive.js

```diff
@@ -208,13 +226,54 @@ define(['angularAMD', 'userSettingsService'], function (angularAMD) {
         }
       }
 
-      function toggleNextTest(currentTest) {
-        $(currentTest).nextAll('.aside-report:not(.is-hidden)')
-        .first()
-        .find('.test-name')
-        .click();
+      function toggleNextTestItem(currentTest) {
+        $(currentTest).nextAll(
+          '.aside-report:not(.is-hidden)')
+          .first()
+          .find('.test-url')
+          .first()
+          .click();     
       }
 
+      function togglePrevTestItem(currentTest) {
+        $(currentTest).prevAll(
+          '.aside-report:not(.is-hidden)')
+          .first()
+          .find('.test-url')
+          .last()
+          .click();
+      }
+
+      function clickOnTab() {
+        var currentTab = userSettingsService.getLastTab();
+        var nextTestTabs = $('.nav-tabs').children();
+        for(var i=0;i <nextTestTabs.length; i++) {
```

Comment:
Why you're not using jQuery `each` for keeping it simpler and more consistent?

Reply:
Could you also fix formatting? Thanks.